vmm/src/vm.rs

```diff
@@ -984,50 +1001,16 @@ impl Vm {
         let new_state = VmState::Running;
         current_state.valid_transition(new_state)?;
 
-        // On x86_64, load_kernel() invokes configure_system().
-        // But on aarch64, it only loads kernel, configure_system()
-        // need to be postponed after VCPU's are created.
-        let entry_addr = self.load_kernel()?;
+        let entry_point = self.load_kernel()?;
 
         // create and configure vcpus
         self.cpu_manager
             .lock()
             .unwrap()
-            .create_boot_vcpus(entry_addr)
+            .create_boot_vcpus(entry_point)
             .map_err(Error::CpuManager)?;
 
-        //
-        // The commented code below illustrates why we have to separate
-        // VCPU's creation and activation into 2 stages to cover aarch64.
-        // (It will be de-commented when supporting code is ready.)
-        //
-        // On aarch64, VCPU's have to be created in prior of GIC.
-        //
-        // #[cfg(target_arch = "aarch64")]
-        // let boot_vcpus = self.cpu_manager.lock().unwrap().boot_vcpus();
-        //
-        // #[cfg(target_arch = "aarch64")]
-        // self.device_manager
-        //     .lock()
-        //     .unwrap()
-        //     .create_irqchip(boot_vcpus)
-        //     .unwrap();
-        //
-        // Once GIC created, register irqfd.
-        // Counterpart of X86 was done in IOAPIC setup.
-        // #[cfg(target_arch = "aarch64")]
-        // self.device_manager
-        //     .lock()
-        //     .unwrap()
-        //     .enable_interrupt_controller()
-        //     .unwrap();
-        //
-
-        // On aarch64, configure_system() is impossible until VCPU's and GIC
-        // created. Because configure_system() will also create FDT which
-        // requires information of VCPU's and GIC.
-        #[cfg(target_arch = "aarch64")]
-        self.configure_system()?;
```

Comment:
Don't add this whole comment in your first commit if you remove it from your second one.